pro-plan-for-free.js

```diff
@@ -9,4 +9,5 @@ module.exports = [
   'itspugle', // 2018-10-24: Student creating tools to better enable volunteer groups,
   'urbanengine', // 2019-03-29: non-profit organization helping startups & small businesses network and grow within their community
   'moonsmile' // 2019-9-1: Student writing tools to learn, I promise i will pay when I finish my project.
```

Comment:
```suggestion
   'moonsmile', // 2019-9-1: Student writing tools to learn, I promise i will pay when I finish my project.
```

Comma is missing and throws a syntax error

Reply:
Added